pkg/scaffold/ansible/galaxy_init.go

```diff
@@ -0,0 +1,42 @@
+package ansible
+
+import (
+	"io/ioutil"
+	"path/filepath"
+
+	"github.com/operator-framework/operator-sdk/pkg/scaffold/input"
+)
+
+// GalaxyInit - wrapper
+type GalaxyInit struct {
+	input.Input
+
+	Kind string
+	Dir  string
+}
+
+// GetInput - get input
+func (g *GalaxyInit) GetInput() (input.Input, error) {
+	if g.Path == "" {
+		dir, err := ioutil.TempDir("", "osdk")
+		if err != nil {
+			return g.Input, err
```

Comment:
`return Input{}, err`

Reply:
Why is the empty input preferable to the g.Input?

Reply:
When returning an error you shouldn't return other computed data. `g.Input` fields might have been set by a previous call to `GetInput()` or modified elsewhere. Returning an empty `Input{}` is cleaner.